native.js

```diff
@@ -11,12 +11,15 @@ import App from './src/mainApp/AppHolder'
 import { name as appName } from './app.json'
 import 'react-native-gesture-handler'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
+import codePush from "react-native-code-push";
```

Comment:
1. create `lib/hoc/withHotCodePush.native.js`

2. declare the following HoC inside

```javascript
import codePush from "react-native-code-push"

const defaultOptions = { checkFrequency: codePush.CheckFrequency.MANUAL }

const withHotCodePush = (component, options = defaultOptions) => codePush(options)(component)

export default withHotCodePush

```

3. re-use it from `native.js`

```javascript
import withHotCodePush  from 'lib/hoc/withHotCodePush'

// ...

const DeApp = withHotCodePush(() => (
   <SafeAreaProvider>
     <WebviewCrypto />
    <App />
  </SafeAreaProvider>
))

// ...
```